backend/internal/models/feedPost.go

```diff
@@ -0,0 +1,23 @@
+package models
+
+import (
+	"time"
+)
+
+type FeedPost struct {
+	ID             int       `json:"id"`
+	UserID         string    `json:"user_id"`
+	Username       string    `json:"username"`
+	ProfilePicture string    `json:"profile_picture"`
+	MediaType      MediaType `json:"media_type"`
+	MediaID        int       `json:"media_id"`
+	Rating         int       `json:"rating"`
```

Comment:
can we add Comment too? representing the actual text associated with the review? this is potentially null

Reply:
Oh yes, totally missed that